src/Me/Me.tsx

```diff
@@ -14,26 +14,24 @@ import { isSsr } from '../helpers/ssr';
 import { useUser } from '../context/userContext/UserContext';
 import { useAuth } from '../context/authContext/AuthContext';
 
-
-
 const Me = (props: any) => {
-  const { children, title } = props
+  const { children, title } = props;
   const { pathname } = useRouter();
-  const {currentUser} = useUser()
-  const auth = useAuth()
+  const { currentUser, isLoading } = useUser();
+  const auth = useAuth();
 
   React.useEffect(() => {
-    if (!currentUser) {
-      auth.login(pathname)
+    if (!isLoading && !currentUser) {
```

Comment:
Don'y login before we finished the query the auth service and the user is undefined